src/logic/useReading.tsx

```diff
@@ -37,14 +42,14 @@ function useProvideReading() {
   useEffect(() => {
     serialDataHandler.startUsbListener();
     console.log('starting generator');
-    // dummyGenerator.startGenerating();
```

Comment:
the changes to uncomment out dummyGenerator should not be included in your code, you can keep them local for testing

Reply:
okay got it.